samples/FSharpReminders/FSharp.Orleans.Reminders/FSharp.Orleans.Reminders.Grains/FSharp.Orleans.Reminders.Grains.fsproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Compile Include="IReminderGrain.fs" />
+        <Compile Include="ReminderGrain.fs" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Orleans.Core.Abstractions" Version="3.6.5" />
```

Comment:
Thank you for the contribution! Is there a chance you could upgrade this sample to Orleans 7.0.0?

Reply:
Of course. Leave it with me

Reply:
Let us know if you hit any speed bumps.
The assembly-level `[assembly: GenerateCodeForDeclaringAssembly(Type)]` attribute can be used to instruct the code generator to examine your F# asm from a C# project

Reply:
Apologies for the delay on this. Trying to find the time too make this change, albeit small

Reply:
Hi @ReubenBond , I am having a little trouble with the upgrade. I believe I have made the relevant changes, the app builds but when it starts, I get the following error

 System.AggregateException: One or more errors occurred. (Could not find an implementation for interface Orleans.IReminderTableGrain)

I noticed this issue that was triaged [Could not find an implementation for interface Orleans.IReminderTableGrain #8158](https://github.com/dotnet/orleans/issues/8158). 

Here's my repo that I have being testing the upgrade on. Wondering if it is possible that the issue is on my machine

[FSharp-Orleans-Reminders/upgrade_to_7](https://github.com/WesleySkeen/FSharp-Orleans-Reminders/tree/upgrade_to_7)

Reply:
@ReubenBond just bumping this one

Reply:
@WesleySkeen @ReubenBond I tried the sample on my machine and it is still giving me the "Could not find an implementation for interface Orleans.IReminderTableGrain" error.

I deleted the .nuget cache and rebuilt the project but still the same error.

It seems DI is not able to hook in the service for IReminderTableGrain under some conditions